ingestion/src/metadata/utils/avro_parser.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021 Collate
```

Comment:
Instead of putting all these in utils, let's create a module `parsers`. We should try to start cleaning things out from `utils` as much as possible into some names that have an actual meaning 😬

Reply:
Should I create the it in the same level as the utils directory?

Reply:
Created it in the same level as utils directory